src/views/components/comment-item/comment-item.js

```diff
@@ -34,29 +79,88 @@ export class CommentItem extends Component {
 
   renderHeader(t, comment) {
     if (!comment.creator) return;
-    const { creator } = comment;
+    const {creator} = comment;
     const avatar = creator.photoURL ? <Img className='avatar' src={creator.photoURL} alt={comment.creator.name}/> : '';
     return (
-      <div className='comment-item-creator'>
-        <span>{ avatar } { creator.name } <Moment locale={t('lang')} unix fromNow>{comment.created.seconds}</Moment></span>
+      <div>
+        {this.renderTooltip(t)}
+        <div className='comment-item-creator'>
+          <span>{avatar} {creator.name} <Moment locale={t('lang')} unix fromNow>{comment.created.seconds}</Moment></span>
+        </div>
       </div>
     );
   }
 
-  renderBody(comment) {
+  renderTooltip(t) {
+    if (this.shouldDisplayTooltip()) {
```

Comment:
As a way to get to cleaner code - I prefer to reduce maximum nesting level - so inverting this - 

https://stackoverflow.com/questions/268132/invert-if-statement-to-reduce-nesting

Here is some more read about it buried somewhere here :) 
https://medium.com/@huytrongnguyen1985/lessons-learnt-from-the-clean-code-robert-c-martin-cecbe2b09139